src/main/scala/scalaz/stream/writer.scala

```diff
@@ -0,0 +1,20 @@
+package  scalaz.stream
+
+import scalaz.\/._
+
+
+object writer {
+
+  /** Promote a `Process` to a `Writer` that writes nothing. */
+  def lift[F[_], A](p: Process[F, A]): Writer[F, Nothing, A] =
```

Comment:
Wait, @alissapajer do you have these flipped - shouldn't `liftO` return the `Writer[F, Nothing, A]` and `liftW` return a `Writer[F, A, Nothing]`?